.github/CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ For getting in touch with the developers, join our public [RocketChat][rocketcha
 * __SECURITY__: Report any potential security bug to us via [our HackerOne page](https://hackerone.com/owncloud) or security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
 * This repository ([web](https://github.com/owncloud/web/issues)) is *only* for issues within the Web code.
 * The issues in other components should be reported in their respective repositories: 
-  - [ownCloud X Server](https://github.com/owncloud/core/issues)
-  - [Infinite Scale server](https://github.com/owncloud/ocis/issues)
+  - [ownCloud Core](https://github.com/owncloud/core/issues)
```

Comment:
```suggestion
  - [ownCloud Classic](https://github.com/owncloud/core/issues)
```
maybe? 🤔

Reply:
what about ownCloud OG 🤣 , joking aside, i picked it because core is the repo name also also called that in the readme, https://github.com/owncloud/core/blob/883a529a8115911794d263d796cecb34bfde8c95/README.md?plain=1#L1.

But i think classic could be a better fit in terms of communication, ill update it